actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -282,8 +285,11 @@ def image_alt(src)
       #   video_tag(["trailer.ogg", "trailer.flv"], size: "160x120")
       #   # => <video height="120" width="160"><source src="/videos/trailer.ogg" /><source src="/videos/trailer.flv" /></video>
       def video_tag(*sources)
```

Comment:
It's turtles all the way down, other method signatures that this gets passed to use extract_options.

Better to do that in a separate PR.